docs/management/advanced-options.asciidoc

```diff
@@ -8,6 +8,7 @@ for displayed decimal values.
 . Go to *Management > {kib} > Advanced Settings*.
 . Scroll or search for the setting you want to modify.
 . Enter a new value for the setting.
+. Click *Save*.
```

Comment:
Right now it's aligned with what we currently have on advanced settings. 

After centralizing the state (https://github.com/elastic/kibana/pull/53693) the message in bottom bar will be *Save changes*. 
<img width="267" alt="Screenshot 2020-02-14 at 16 56 31" src="https://user-images.githubusercontent.com/4283304/74546531-f8e2aa80-4f4a-11ea-94e2-5c6e671ea704.png">
Do you think it makes sense to add the word 'Changes'?